### PR TITLE
#633-webui Drop the validation rule from the Receipt Schedule's destination warehouse

### PR DESCRIPTION
**Pull request by metas-rc**
https://github.com/metasfresh/metasfresh-webui-api/issues/633

### CODE REVIEW

**Comment by teosarca** [comment]
note: check https://github.com/metasfresh/metasfresh-webui-api/issues/633#issuecomment-338587735

There are 3 tasks around this topic.  I think we shall come up with a common solution.

**Comment by metas-ts** [comment]
Hi @metas-rc i assigned the PR to you for now
please unassign from yourself when you think it's ready for R&I

File: de.metas.adempiere.adempiere/client/src/main/java/org/compiere/grid/ed/VLookupAutoCompleterValidationRule.java [hunk 1]
```diff
@@ -89,10 +91,22 @@ public IStringExpression getPrefilterWhereClause()
...
+	@Override
+	public List<ValueNamePair> getExceptionTableAndColumns()
+	{
+		return ImmutableList.of();
+	}
+
+	@Override
+	public void registerException(final String tableName, final String columnName)
+	{
+		throw new NotImplementedException("There is no implementation for registering esceptions in the VLookupAutoCompleterValidationRule class: " + this);
```
Comment (metas-ts) [inline de.metas.adempiere.adempiere/client/src/main/java/org/compiere/grid/ed/VLookupAutoCompleterValidationRule.java]: briefly thought with teo about this: `NotImplementedException` is a much nicer classname that `UnSupportedOperationException`, but `UnSupportedOperationException` is in the `java.lang` package and i'm not sure we want to further bind ourselves to commons-lang3..

File: de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/GridFieldVO.java [hunk 2]
```diff
@@ -791,8 +792,11 @@ private void createLookupInfo(final boolean alwaysCreate)
 				}
 				else
 				{
+					final String tablename = Services.get(IADTableDAO.class).retrieveTableName(getAD_Table_ID());
 					lookupInfo = MLookupFactory.getLookupInfo(WindowNo, displayType,
-							ColumnName, AD_Reference_Value_ID,
+							tablename, //tableName
```
[the remark this answers is not in the document]
Comment (metas-rc) [reply de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/GridFieldVO.java]: Tobi fixed this one and committed the change so it is no longer in my branch,.

File: de.metas.adempiere.adempiere/client/src/main/java/org/compiere/grid/ed/VLookupAutoCompleterValidationRule.java (hunk 1 above)
Comment (metas-rc) [reply de.metas.adempiere.adempiere/client/src/main/java/org/compiere/grid/ed/VLookupAutoCompleterValidationRule.java]: Done

Comment (metas-rc) [reply de.metas.adempiere.adempiere/client/src/main/java/org/compiere/grid/ed/VLookupAutoCompleterValidationRule.java]: Done

**Pull request merged**